h/features/client.py

```diff
@@ -79,4 +79,7 @@ def _state(self, feature):
         # member.
         if feature.staff and role.Staff in self.request.effective_principals:
             return True
+        for cohort in feature.cohorts:
```

Comment:
This would probably be clearer (and would definitely be more efficient) if written as:

```python
if set(feature.cohorts) & set(self.request.authenticated_user.cohorts):
    return True
```